### ISSUE TITLE
Release parquet-mr 1.12.0

### ISSUE THREAD

**Issue by asfimport** [apache/parquet-java#2516]
This task is to track the parquet-mr release 1.12.0.

Any open Jira that we would like to be added to this release shall be linked to this one as a blocker.

**Reporter**: [Gabor Szadovszky](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=gszadovszky) / @gszadovszky
**Assignee**: [Gabor Szadovszky](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=gszadovszky) / @gszadovszky
#### Related issues:
- [Error: SYSTEM ERROR: RuntimeException: Unknown logical type <LogicalType UUID:UUIDType()>](https://issues.apache.org/jira/browse/DRILL-7825) (blocks)
- [Fix build with JDK11 for JDK8](https://issues.apache.org/jira/browse/PARQUET-1966) (is blocked by)
- [Make minor releases source compatible](https://issues.apache.org/jira/browse/PARQUET-1970) (is blocked by)
- [Flaky test in github action](https://issues.apache.org/jira/browse/PARQUET-1971) (is blocked by)
- [Invalid data_page_offset](https://issues.apache.org/jira/browse/PARQUET-1977) (is blocked by)
- [Optional bloom_filter_offset is filled if no bloom filter is present](https://issues.apache.org/jira/browse/PARQUET-1979) (is blocked by)

<sub>**Note**: *This issue was originally created as [PARQUET-1898](https://issues.apache.org/jira/browse/PARQUET-1898). Please see the [migration documentation](https://issues.apache.org/jira/browse/PARQUET-2502) for further details.*</sub>

**Comment by asfimport** [comment]
[pavithra ramachandran](https://issues.apache.org/jira/browse/PARQUET-1898?#comment-17261224):
When can we expect parquet 1.12.0 release?

**Comment by asfimport** [comment]
[Gabor Szadovszky](https://issues.apache.org/jira/browse/PARQUET-1898?#comment-17261290) / @gszadovszky:
`[~pavithraramachandran]`, we still have a couple of open PRs that are targeted to 1.12.0 and the community does not seem to be very active reviewing/committing them. After these are in I'll initiate an RC for the release.

**Comment by asfimport** [comment]
[Charles Givre](https://issues.apache.org/jira/browse/PARQUET-1898?#comment-17267990):
`[~pavithraramachandran]` What PRs are blockers?   Can we link them?

**Comment by asfimport** [comment]
[Vitalii Diravka](https://issues.apache.org/jira/browse/PARQUET-1898?#comment-17300373) / @vdiravka:
Looks like all "is blocked by" tickets are resolved and there is only one opened task with fixVersion=1.12, this is this current task:
```java

"project = PARQUET AND status in (Open, "In Progress", Reopened, "Patch Available") AND fixVersion = 1.12.0"
```
What PRs are targeted to these release?
https://github.com/apache/parquet-mr/pulls

**Comment by asfimport** [comment]
[Gabor Szadovszky](https://issues.apache.org/jira/browse/PARQUET-1898?#comment-17300430) / @gszadovszky:
@vdiravka, the release is ongoing. The [RC3 vote](https://lists.apache.org/thread.html/r3668cc61190a0c0d94d2f819589f5e2ab92f7699b0f66809b109d9bf%40%3Cdev.parquet.apache.org%3E) is started today. Feel free to subscribe to the dev list and/or verify the RC.
